Rebus.AdoNet.Tests/SagaPersisterTests.cs

```diff
@@ -874,34 +1056,29 @@ public void NoChangesAreMadeWhenUoWIsNotCommitted()
 		public void FindSameSagaTwiceThrowsOnNoWait()
 		{
 			// Skip this test on sqlite, as it does not support locking.
-			if (ProviderName == GetSqliteProviderName())
+			if (!_features.HasFlag(Feature.NoWait))
 			{
-				Assert.Ignore("Test not supported on sqlite database.");
+				Assert.Ignore("Test not supported when NoWait Locking is disabled or unsupported.");
 				return;
 			}
 
-			var persister = CreatePersister(createTables: true, useLocking: true, useNoWaitLocking: true);
+			var persister = _persister;
 			var savedSagaData = new MySagaData();
 			var savedSagaDataId = Guid.NewGuid();
 			savedSagaData.Id = savedSagaDataId;
 			persister.Insert(savedSagaData, new string[0]);
 
 			using (_manager.Create())
+			using (_manager.GetScope(autocomplete: true))
```

Comment:
Why is this needed?

Reply:
lmao this is yours xd

Reply:
Then it may have been needed by Yuga.. or I left this by mistake after completing yuga impl. 

Can you test wether the test still passes (under psql + yuga) w/o this line?

Reply:
Working against postgres & yuga without it